### ISSUE TITLE
`bowling.py` depracted

### ISSUE THREAD

**Issue by skhrg** [MUSTANG-SZ/WITCH#77]
Don't think anything in `bowling.py` is used anywhere anymore, should we just delete it?

Pinging @Sulla2012 on this.

**Comment by Sulla2012** [comment]
It's sometimes nice for plotting but should absolutely not be used for fitting so yeah I think this should be deprecated.

**Comment by skhrg** [comment]
Could just move it to the scratch dir so we still have it.

**Comment by Sulla2012** [comment]
Yeah I think that makes sense. Ultimately I'd like to refactor our plotting tools and I can wrap that in to that refactor so moving it to scratch is a good temporary choice.
